encryption.go

```diff
@@ -143,6 +143,9 @@ func EncryptLayer(ec *config.EncryptConfig, encOrPlainLayerReader io.Reader, des
 
 		newAnnotations := make(map[string]string)
 		keysWrapped := false
+		if len(keyWrapperAnnotations)==0{
```

Comment:
This is not properly gofmt'ed. Can you run `gofmt -w -s encryption.go` on this file?

Can you put a sentence into the commit describingwhether you ran into a problem with missing keyWrapperAnnotations or whether you added the check primarily because it's missing?

Thanks.